app/src/main/java/com/gathering/android/event/home/EventListViewModel.kt

```diff
@@ -134,6 +174,16 @@ class EventListViewModel @Inject constructor() : ViewModel() {
             )
         )
     }
+
+    private fun getDistance(locationA: Location, locationB: Location): Double {
+        val dx = locationB.longitude - locationA.longitude
+        val dy = locationB.latitude - locationA.latitude
+        return sqrt(dx * dx + dy * dy)
+    }
+
+    private fun getCurrentLocation(): Location {
```

Comment:
lets take this method to a new class called `LocationProvider` and pass a instance of it in `EventLocationComperator` class as a dependency.

Reply:
it will be addressed in T29